keps/sig-storage/2268-non-graceful-shutdown/README.md

```diff
@@ -349,27 +368,32 @@ previous answers based on experience in the field._
     (e.g. update of object X triggers new updates of object Y)
   - periodic API calls to reconcile state (e.g. periodic fetching state,
     heartbeats, leader election, etc.)
+  No.
```

Comment:
Generally I put no against each question.

Reply:
Doesn't this result pod deletion and volume detatch calls?